kuma/users/jinja2/users/ban_user_and_cleanup_summary.html

```diff
@@ -41,7 +41,41 @@
       {% else %}
       <p>{{ _('The user did not have any revisions that were reported as spam.') }}</p>
       {% endif %}
-    </div>
+    </div><!-- /#revisions-reported-as-spam -->
+
+    <h5>{{ _('Deleted:')}}</h5>
+    <div id="revisions-deleted">
+      {% if actions_taken.revisions_deleted_list %}
+        <ul>
+          {% for revision in actions_taken.revisions_deleted_list %}
+            <li>
+              <a href="{{ revision.document.get_absolute_url() }}">
+                {{ revision.document.title }}
+              </a>
+            </li>
+          {% endfor %}
+        </ul>
+      {% else %}
+      <p>{{ _('The user did not have any revisions that were deleted.') }}</p>
+      {% endif %}
+    </div><!-- /#revisions-deleted -->
+
+    <h5>{{ _('Reverted:')}}</h5>
+    <div id="revisions-reverted">
+      {% if actions_taken.revisions_reverted_list %}
+        <ul>
+          {% for revision in actions_taken.revisions_reverted_list %}
+            <li>
+              <a href="{{ revision.get_absolute_url() }}">
```

Comment:
Thank you for adding the link. In manual testing, I found that sometimes I wanted to see the document as it is now, and other times I wanted to see the document history.  I'd want the "View Page" and "History" buttons from the "Revisions already identified as spam" section. This can wait until @stephaniehobson does the front-end review, though, to see if she agrees.